### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by wojgp**

### CODE REVIEW

**Review by wojgp — COMMENTED** [review]
Take a look

File: readme.md [hunk 1]
```diff
@@ -12,25 +12,31 @@ SnowGem node AMI app.
...
+            "masternode": 1,
+            "rpcallowip": "127.0.0.0/8",
+            "masternodeprivkey": "5JM..........QYhRN"
         }
-    },
-    "user": "xsg1", 
-}
-```
+    }
+    "user": "xsg1",
```
Comment (cryi) [inline readme.md]: move coma to the line above

File: readme.md [hunk 2]
```diff
@@ -12,25 +12,31 @@ SnowGem node AMI app.
...
+}```
+Above example is a config file for 22nd masternode. Depending on the node You have to change these lines:
+``` "id": "xsg22",
+ "bind": "aaa.bbb.ccc.ddd",
+ "rpcbind": "127.0.0.22",
+ "masternodeprivkey": "5JM..........QYhRN"```
```
Comment (cryi) [inline readme.md]: move \`\`\` to new line

**Review by cryi — CHANGES_REQUESTED** [review]
Please fix issues I commented. Other than that looks alright

**Comment by cryi** [comment]
Good job. Thank you

**Pull request merged**